python/rikai/types/geometry.py

```diff
@@ -392,6 +395,25 @@ def iou(
             return iou_arr[0]
         return iou_arr
 
+    def with_label(
+        self, text: str, color: str = get_option(CONF_RIKAI_VIZ_COLOR)
```

Comment:
Nits:  usually python will take `color: Option[str] = None`, and within the function itself, try to check 

```
if color is None:
    color = get_option(...)
```

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/3381789/163900921-9b5faa8a-14e2-43d4-9a88-155016fc24e0.png">
I see the current pattern many times in the master codebase, should we change them all? I chose the current way that's because I want to be consistent with existing code base.

Reply:
I agree with Eddy, and we should change them all. But I think it should be done in a separate pull request.